conf/messages

```diff
@@ -959,14 +959,26 @@ previousYearsLossesValue.error.required=Enter an amount of Capital Gains Tax los
 previousYearsLossesValue.label=Amount of losses from previous tax years
 previousYearsLosses.link=<a href="{0}" target="_blank">Find out how to report losses (opens in a new window)</a>.
 
-annualExemptAmount.title=How much of your Capital Gains Tax Annual Exempt Amount do you want to use?
-annualExemptAmount.helpText=In the tax year {0} to {1}, you only have to pay Capital Gains Tax on gains above {2}.
+annualExemptAmount.individual.title=How much of your Capital Gains Tax Annual Exempt Amount do you want to use?
+annualExemptAmount.trust.title=How much of the trust’s Capital Gains Tax Annual Exempt Amount does it want to use?
+
+annualExemptAmount.individual.helpText=In the tax year {0} to {1}, you only have to pay Capital Gains Tax on gains above {2}.
+annualExemptAmount.trust.helpText=In the tax year {0} to {1}, most trustees only have to pay Capital Gains Tax on gains above £6,150. This is called the Annual Exempt Amount. <br/> <br/>The Annual Exempt Amount for trusts for vulnerable beneficiaries is £12,300.
```

Comment:
we've still got `£6150` and `£12,300` not being passed in as parameters. You can find these in the `TaxYear` in the disposal date under the annual exempt amounts (see here: https://github.com/hmrc/cgt-property-disposals/blob/master/conf/tax_years.conf#L36)